websites/G/Google Translate/presence.ts

```diff
@@ -254,7 +266,7 @@ presence.on("UpdateData", async () => {
   const showType: boolean = await presence.getSetting("type");
 
   if (showType) {
-    presenceData.details = `Translating: ${typet}`;
+    presenceData.details = `Translating: ${type}`;
     presenceData.state = `From: ${languageCode(from)} - To: ${languageCode(
```

Comment:
I would recommend removing the colons and the dash here to make it more readable